rfcs/text/0000-dss-metrics.md

```diff
@@ -0,0 +1,65 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# RFC: Key performance metrics of the DCP Data Store (DSS)
+
+## Summary
+
+This RFC proposes a set of key performance metrics to be introduced and tracked as a summary measure of performance by
+which the utility of the Data Store can be evaluated, including improvements and regressions.
+
+## Author(s)
+
+* [Andrey Kislyuk](mailto:akislyuk@chanzuckerberg.com)
+
+## Shepherd
+***Leave this blank.** This role is assigned by DCP PM to guide the **Author(s)** through the RFC process.*
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## Motivation
+
+Beyond latency and error rates, the DSS lacks a clear focus on quantifiable assessment of usability. We would like to
```

Comment:
I would like a better understanding of what these metrics are (is latency per API call? what are the units of the error rates? etc.) as well as a brief description on how they are tracked today. Maybe this could be added to the Design section.